conf/application.conf

```diff
@@ -92,6 +92,7 @@ microservice {
     features {
       welsh-translation = false
       use-improved-error-messages = false
+      ead-document = true
```

Comment:
AFAIR for ILE Query we use values enabled/disabled. Could we stay with one approach?
It seems that we may have more feature flags in the future, so it'd be better to be aligned. Not thinking every time "should I write true or enabled?". 
And not enforcing consistency may lead to even more approaches.

Reply:
I sticked with the true or false, because of the above configs use the same pattern. we're not using enabled disabled in this application. Good point, but if you're happy with it I'll stay with this configuration for the time being

Reply:
Changed to enabled disabled